packages/vscode-extension/src/webview/utilities/consts.ts

```diff
@@ -26,6 +25,7 @@ export type DeviceProperties = {
   maskImage: string;
 };
 
+// When including a new device update the `supportedDevices.ts` in extension with the model identifier.
```

Comment:
not sure what you mean here, this doesn't seem to be a grammatically correct sentence